Content.Server/Administration/Commands/OSay.cs

```diff
@@ -0,0 +1,63 @@
+using System.Linq;
+using Content.Server.Administration.Logs;
+using Content.Server.Chat.Systems;
+using Content.Shared.Administration;
+using Content.Shared.Database;
+using Robust.Shared.Console;
+
+namespace Content.Server.Administration.Commands
+{
```

Comment:
```suggestion
namespace Content.Server.Administration.Commands;
```
Namespaces should be file-scoped for new files, even if it's inconsistent with other files in that directory.